engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -139,8 +139,11 @@ def __init__(self, *args, **kwargs) -> None:
         self._neuronx_class = self.set_neuronx_class()
 
     def set_neuronx_class(self):
-        module_name, class_name = self.MODEL_TYPE_TO_CLS_LOADER[
-            self.model_config.model_type].rsplit(".", maxsplit=1)
+        try:
+            module_name, class_name = self.MODEL_TYPE_TO_CLS_LOADER[
+                self.model_config.model_type].rsplit(".", maxsplit=1)
+        except KeyError as exc:
+            raise KeyError(f"Unsupported model: {str(exc)}")
```

Comment:
We should also return the list of model types that are supported.

Reply:
Added: https://github.com/deepjavalibrary/djl-serving/blob/2f131026b30d1c4ea72641b395e2a1b0b5341b14/engines/python/setup/djl_python/neuron_utils/model_loader.py#L146-L148

Example output:
```
OptimizationFatalError('Partitioning exited with return code: 1. Details: KeyError: "Unsupported model: 'RefinedWebModel'. Supported architectures: ['opt', 'gpt2', 'gptj', 'gpt_neox', 'llama', 'mistral', 'mixtral', 'bloom']"')
```